src/content/guides/how-to-revoke-token-access/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: How to revoke smart contract access to your crypto funds
+description: A how to guide on revoking exploitative smart contract token access
+lang: en
+---
+
+# How to: Revoke smart contract access to your crypto funds
+
+With this guide you will learn how to view a list of all smart contracts you have allowed access to your funds and also how to cancel them. 
```

Comment:
```suggestion
This guide will teach you how to view a list of all smart contracts you have allowed access to your funds and how to cancel them.
```

Reply:
Also, 'access to your funds' or 'access to your account' 🤔 it gets muddy because you can grant access to only specific tokens I guess...